internal/provider/provider.go

```diff
@@ -34,6 +34,11 @@ type AdaptiveMetricsProvider struct {
 	// provider is built and ran locally, and "test" when running acceptance
 	// testing.
 	version string
+
+	// commit is set to the provider commit on release, "unknown" when the
+	// provider is built and ran locally, and "unknown" when running acceptance
```

Comment:
As a non-native english speaker, this strikes me as a strange way to construct the sentence. If two conditions result in it being `unknown` shouldn't the sentence be something like:

```suggestion
	// commit is set to the provider commit on release, "unknown" when the
	// provider is built and ran locally or when running acceptance
```

With the current sentence I first thought there was a mistake and in one of these two cases the value should be something different.

Reply:
Fair point, thanks!